<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>640</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>80</red>
        <green>87</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>80</red>
        <green>87</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>80</red>
        <green>87</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>80</red>
        <green>87</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="primaryArchetypes">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Primary Archetype</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="secondaryArchetypes">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>180</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Secondary Archetype</string>
     </property>
    </item>
   </widget>
   <widget class="QScrollArea" name="archetypeDescSelect">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>248</red>
          <green>255</green>
          <blue>220</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>248</red>
          <green>255</green>
          <blue>220</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>80</red>
          <green>87</green>
          <blue>127</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>219</width>
       <height>59</height>
      </rect>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>0</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Unselect Archetypes</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTextEdit" name="archetypeDesc">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>230</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="startProgram">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>500</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 16pt &quot;Rockwell Extra Bold&quot;;
background-color: rgb(28, 113, 2);
border: 2px solid #555;
border-radius: 20px;
border-style: outset;</string>
    </property>
    <property name="text">
     <string>Start Program</string>
    </property>
   </widget>
   <widget class="QPushButton" name="advancedFeatures">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>560</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Advanced Features â–º</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>341</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>541</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="abilities">
     <item>
      <layout class="QVBoxLayout" name="leftColumn">
       <item>
        <layout class="QHBoxLayout" name="strLayout">
         <item>
          <widget class="QLabel" name="strName">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Nirmala UI&quot;;</string>
           </property>
           <property name="text">
            <string>Strength</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="strLowestVal">
           <property name="minimum">
            <number>8</number>
           </property>
           <property name="maximum">
            <number>17</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="strTo">
           <property name="text">
            <string>to</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="strHighestVal">
           <property name="minimum">
            <number>8</number>
           </property>
           <property name="maximum">
            <number>17</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="intLayout">
         <item>
          <widget class="QLabel" name="intName">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Nirmala UI&quot;;</string>
           </property>
           <property name="text">
            <string>Intelligence</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="intLowestVal">
           <property name="minimum">
            <number>8</number>
           </property>
           <property name="maximum">
            <number>17</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="intTo">
           <property name="text">
            <string>to</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="intHighestVal">
           <property name="minimum">
            <number>8</number>
           </property>
           <property name="maximum">
            <number>17</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="midColumn">
       <item>
        <layout class="QHBoxLayout" name="dexLayout">
         <item>
          <widget class="QLabel" name="dexName">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Nirmala UI&quot;;</string>
           </property>
           <property name="text">
            <string>Dexterity</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="dexLowestVal">
           <property name="minimum">
            <number>8</number>
           </property>
           <property name="maximum">
            <number>17</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="dexTo">
           <property name="text">
            <string>to</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="dexHighestVal">
           <property name="minimum">
            <number>8</number>
           </property>
           <property name="maximum">
            <number>17</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="wisLayout">
         <item>
          <widget class="QLabel" name="wisName">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Nirmala UI&quot;;</string>
           </property>
           <property name="text">
            <string>Wisdom</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="wisLowestVal">
           <property name="minimum">
            <number>8</number>
           </property>
           <property name="maximum">
            <number>17</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="wisTo">
           <property name="text">
            <string>to</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="wisHighestVal">
           <property name="minimum">
            <number>8</number>
           </property>
           <property name="maximum">
            <number>17</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="rightColumn">
       <item>
        <layout class="QHBoxLayout" name="conLayout">
         <item>
          <widget class="QLabel" name="conName">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Nirmala UI&quot;;</string>
           </property>
           <property name="text">
            <string>Constitution</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="conLowestVal">
           <property name="minimum">
            <number>8</number>
           </property>
           <property name="maximum">
            <number>17</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="conTo">
           <property name="text">
            <string>to</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="conHighestVal">
           <property name="minimum">
            <number>8</number>
           </property>
           <property name="maximum">
            <number>17</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="chaLayout">
         <item>
          <widget class="QLabel" name="chaName">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Nirmala UI&quot;;</string>
           </property>
           <property name="text">
            <string>Charisma</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="chaLowestVal">
           <property name="minimum">
            <number>8</number>
           </property>
           <property name="maximum">
            <number>17</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="chaTo_2">
           <property name="text">
            <string>to</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="chaHighestVal">
           <property name="minimum">
            <number>8</number>
           </property>
           <property name="maximum">
            <number>17</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>410</y>
      <width>561</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="skills">
     <item>
      <layout class="QVBoxLayout" name="firstColumn">
       <item>
        <widget class="QCheckBox" name="checkBox_28">
         <property name="text">
          <string>Acrobatics</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_20">
         <property name="text">
          <string>Insight</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_21">
         <property name="text">
          <string>Performance</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="secondColumn">
       <item>
        <widget class="QCheckBox" name="checkBox_25">
         <property name="text">
          <string>Animal Handling</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_26">
         <property name="text">
          <string>Intimidation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_27">
         <property name="text">
          <string>Persuasion</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="thirdColumn">
       <item>
        <widget class="QCheckBox" name="checkBox_19">
         <property name="text">
          <string>Arcana</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_29">
         <property name="text">
          <string>Investigation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_30">
         <property name="text">
          <string>Religion</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="fourthColumn">
       <item>
        <widget class="QCheckBox" name="checkBox_31">
         <property name="text">
          <string>Athletics</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_32">
         <property name="text">
          <string>Medicine</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_33">
         <property name="text">
          <string>Sleight of Hand</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="fifthColumn">
       <item>
        <widget class="QCheckBox" name="checkBox_34">
         <property name="text">
          <string>Deception</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_35">
         <property name="text">
          <string>Nature</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_36">
         <property name="text">
          <string>Stealth</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="sixthColumn">
       <item>
        <widget class="QCheckBox" name="checkBox_37">
         <property name="text">
          <string>History</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_38">
         <property name="text">
          <string>Perception</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_39">
         <property name="text">
          <string>Survival</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
